### DISCUSSION TITLE
Disabling Preflight

### DISCUSSION THREAD

**Discussion by DavidReinertToptal** [ben-rogerson/twin.macro#706] [Q&A]
Hey everyone, just began exploring this library in recent days. I need a runtime solution for TailwindCSS so it can function in a module federation context and I'm optimistic that twin.macro+@emotion/react can be that solution. I did have one question/problem around the preflight tailwindCSS styling. The projects I'm working with currently all expect the preflight to be disabled. Currently. using emotion, I am injecting the `GlobalStyles` from twin.macro as well my own custom global styles, however it seems that pulls in both the preflight styles _and_ other useful or necessary styles such as animation keyframes, without any respect to my [tailwind config setting to disable preflight styles](https://tailwindcss.com/docs/preflight#disabling-preflight)

```.jsx
import * as React from 'react';
import { Global } from '@emotion/react';
import { theme, GlobalStyles as BaseStyles } from 'twin.macro';

const customStyles = {
    html: {
        color: theme`colors.grey.60`,
        fontFamily: `fontFamily.ui`,
        '-webkit-font-smoothing': 'antialiased',
        '-moz-osx-font-smoothing': 'grayscale',
        letterSpacing: '0.25px',
        lineHeight: 1.5,
        margin: 0,
        padding: 0,
        fontSize: '16px',
    },
    body: {
        color: theme`colors.grey.60`,
        fontFamily: theme`fontFamily.ui`,
        '-webkit-font-smoothing': 'antialiased',
        '-moz-osx-font-smoothing': 'grayscale',
        letterSpacing: '0.25px',
        lineHeight: 1.5,
        margin: 0,
        padding: 0,
        fontSize:
            '14px' /* overrides 16px base so text elements like paragraphs inherit 14px instead */,
    },
    '*': {
        boxSizing: 'border-box',
    },
    '*:before': {
        boxSizing: 'border-box',
    },
    '*:after': {
        boxSizing: 'border-box',
    },
};

const GlobalStyles = () => (
    <>
        <BaseStyles />
        <Global styles={customStyles} />
    </>
);

export default GlobalStyles;
```
Does this library currently support my use case or should it? Is this a known issue and could a workaround/support be implemented. I'd be happy to take a stab at it myself if someone would be able to help point me in the right direction. Any insight or guidance would be greatly appreciated. I'm signing off for the evening in my time zone, apologies in advance for any delay in response

**Comment by ben-rogerson** [comment]
Hey there, thanks for raising this issue!
I've jumped along and added support in #707 

```js
// tailwind.config.js
module.exports = {
  corePlugins: {
    preflight: false, // Disable preflight styles in twin 
  }
}
```
